BatteryMeter.c

```
@@ -25,34 +25,142 @@ static const int BatteryMeter_attributes[] = {
+         buf += ret; len -= ret;
+      } else if (info.ac == AC_ABSENT) {
+         ret = xSnprintf(buf, len, "Using bat, ");
+         buf += ret; len -= ret;
+      }
+
+      if (ret && len > 2) {
+         *buf++ = ',';
+         *buf++ = ' ';
+         *buf = 0;
+         len -= 2;
+      }
```

Cppcheck (2.20.0)

[style] The function 'Vector_type' is never used.

(unusedFunction)